code/game/turfs/closed/walls.dm

```diff
@@ -70,8 +70,12 @@
 		if(istype(O, /obj/structure/sign/poster))
 			var/obj/structure/sign/poster/P = O
 			P.roll_and_drop(src)
+	var/turf/nextTurf = ScrapeAway()
+	if(prob(0.0001))
+		var/datum/effect_system/smoke_spread/asbestos/smoke = new
+		smoke.set_up(2, nextTurf)
+		smoke.start()
```

Comment:
this is really unfunny joke that will rarely result in completely unwarranted and almost impossible (unless you have anti tox on hand) to prevent death of many people.
This is hell for admins since the smoke has no logging and it's plainly stupid.